src/Neo.Extensions/ByteArrayComparer.cs

```diff
@@ -0,0 +1,46 @@
+// Copyright (C) 2015-2024 The Neo Project.
+//
+// ByteArrayComparer.cs file belongs to the neo project and is free
+// software distributed under the MIT software license, see the
+// accompanying file LICENSE in the main directory of the
+// repository or http://www.opensource.org/licenses/mit-license.php
+// for more details.
+//
+// Redistribution and use in source and binary forms with or without
+// modifications are permitted.
+
+using System;
+using System.Collections.Generic;
+using System.Runtime.CompilerServices;
+
+namespace Neo.Extensions
+{
+    public class ByteArrayComparer : IComparer<byte[]>
```

Comment:
I don't know why github did't show the previous code

Reply:
git considers that the old code is moved to the benchmark.....